pkg/features/kube_features.go

```diff
@@ -895,18 +895,6 @@ const (
 	// ImageMaximumGCAge enables the Kubelet configuration field of the same name, allowing an admin
 	// to specify the age after which an image will be garbage collected.
 	ImageMaximumGCAge featuregate.Feature = "ImageMaximumGCAge"
-
-	// owner: @saschagrunert
-	//
-	// Enables user namespace support for Pod Security Standards. Enabling this
-	// feature will modify all Pod Security Standard rules to allow setting:
-	// spec[.*].securityContext.[runAsNonRoot,runAsUser]
-	// This feature gate should only be enabled if all nodes in the cluster
-	// support the user namespace feature and have it enabled. The feature gate
-	// will not graduate or be enabled by default in future Kubernetes
-	// releases.
```

Comment:
I worry this is no longer idiomatic but I am following the documented plan. If I should lock it to true and keep it around I'm happy to do so.

Reply:
@jpbetz @liggitt how do you think compatibility versions should intersect with version pinning in pod security admission? The use of feature gates here is a little strange to begin with.

I wonder if we should override `latest` to be the compatibility version if it's set?